tally_ho/apps/tally/models/result_form.py

```diff
@@ -252,6 +252,14 @@ def audit_recommendation(self):
             recomendation_index = self.audit.resolution_recommendation
             return ClearanceResolution.choices()[recomendation_index.value][1]
 
+    @property
+    def audit_quaritine_check_name(self):
+        if self.audit:
+            quarantine_checks = self.audit.quarantine_checks.all()
+            quaritine_check_name =\
```

Comment:
As discussed [above](https://github.com/onaio/tally-ho/pull/271/files#r404991585) there may be multiple different quarantine checks attached to audit and we want to get all of their names, so can join map name over the results or equivalent

Reply:
Done

Reply:
Refactoring the constants

Reply:
Done